### DISCUSSION TITLE
Deselecting an Area

### DISCUSSION THREAD

**Discussion by NoSignalBeep** [maruohon/litematica#755] [General]
I took a schematic in my world for a friend a few days ago and didn't realize the area selection box remained. Days later and thousands of blocks away, I had pasted a schematic I edited into my world and I had to fix a few things again. I went to save and it said saving task created. I went to the coordinates on the HUD, which were around 2000 blocks away and the area selection box is still there. How do I get rid of it? 
Edit: I made a hotkey for delete selection box and nothing is happening.
Edit Two: I just made a temporary fix by moving the selection box to one block right next to the schematic I edited, which I'm assuming didn't save in the edit. I've had an issue with area selection boxes still being in my world and I just make it so they don't render. Is deleting them an immediate action sort of thing?

**Comment by maruohon** [comment]
You can't delete the area selection box if you are in the `Simple` area selection mode.

To remove/hide an area selection box in Litematica, you have 3 options:
* Turn off the area selection box rendering (in Visuals -> `enableAreaSelectionBoxesRendering` or using a hotkey `toggleAreaSelectionBoxesRendering`)
* Switch to the `Normal` selection mode and unselect any open selection file in the Area Selection Browser
* (If you are in `Simple` mode) just collapse it somewhere out of the way (in `Expand` corners mode by right clicking where you want to move it, or in the `Corners` mod by both left and right clicking in the same spot to move both corners there)

I'd say probably the most common of these is to switch to the `Normal`/Multi-Region mode while not having a selection save file open.

Here is a sort of tutorial to the Normal/Multi-Region mode, in case you want to know more about that:
https://github.com/maruohon/litematica/issues/716#issuecomment-1596121219

**Reply by NoSignalBeep** [reply]
Okay, thank you very much. I appreciate the response. Very courteous. Have a wonderful day.
